src/20_Components/amp-facebook-page.html

```diff
@@ -0,0 +1,75 @@
+<!--
+  ## Introduction
+
+  The amp-facebook-page component allows embedding Facebook pages into AMP files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+  <title>amp-facebook-page</title>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <!-- Import the amp-facebook-page component -->
+    <script async custom-element="amp-facebook-page" src="https://cdn.ampproject.org/v0/amp-facebook-page-0.1.js"></script>
+  <link rel="canonical" href="<%host%>/components/amp-facebook-page/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  </head>
+  <body>
+
+    <!-- ## Basic Usage -->
+    <!-- The `amp-facebook-page` component only requires the href of the facebook post... -->
+    <amp-facebook-page width="340" height="130"
+        layout="responsive"
+        data-href="https://www.facebook.com/imdb/">
+    </amp-facebook-page>
+
+    <!-- ## Customizing the embed -->
+    <!-- The `amp-facebook-page` component allows you to customize the embed via several attributes: -->
+
+    <!-- **Example 1: You can set the locale to render the page in a different language, for example French.** -->
```

Comment:
Suggestion: you always prefix subsample with "Example X". I like this approach, however, we don't do this for the other samples. Could you, for consistency reasons, use subheadings instead. For example:

```
 <!-- 
### Different Languages

You can set the locale to render the page in a different language, for example French. 
-->
...
```

This way the different samples show up in the menu to the left.

Reply:
Done.